engines/director/score.cpp

```diff
@@ -1539,6 +1539,10 @@ void Score::startLoop() {
 	_stopPlay = false;
 	_nextFrameTime = 0;
 
+	if (debugChannelSet(-1, kDebugFewFramesOnly)){
```

Comment:
Code formatting is wrong. Missing space before the curly bracket. Or better drop the brackets.